.buildscripts/deploy_snapshot.sh

```diff
@@ -17,9 +18,9 @@ elif [ "$CIRCLE_PROJECT_USERNAME" != "$USERNAME" ]; then
 elif [ "$CIRCLE_JDK_VERSION" != "$JDK" ]; then
   # $CIRCLE_JDK_VERSION must be manually set in circle.yml
   echo "Skipping snapshot deployment: wrong JDK. Expected '$JDK' but was '$CIRCLE_JDK_VERSION'."
-else
+elif [ "$CIRCLE_BRANCH" == "main" ] || [ "$CIRCLE_BRANCH" == "develop" ] && [[ "$VERSION" == *SNAPSHOT ]]; then
```

Comment:
wouldn't the check for `main` and `develop` branches already be covered by the filter in `config.yml`?

Reply:
yes, but we only want to deploy develop when it's a snapshot.

If I remove the check, develop will also develop non snapshots. I guess we can remove the ` "$CIRCLE_BRANCH" == "main" ]` part of the condition and keep the `else` branch. Do you think that's better?

Reply:
oh, right, my bad. Then it might be a good idea to keep it, since it makes the intent pretty explicit